src/test/java/de/unistuttgart/iste/gits/course_service/integration/QueryCourseMembershipsTest.java

```diff
@@ -93,6 +100,77 @@ void testMembership(final GraphQlTester tester){
                 .contains(courseMemberships.get(0), courseMemberships.get(1));
     }
 
+    @Test
+    void testMembershipsFieldInCourse(HttpGraphQlTester tester) {
+        final CourseEntity course = courseRepository.save(createTestCourse());
+
+        final LoggedInUser currentUser = userWithMembershipInCourseWithId(course.getId(),
+                LoggedInUser.UserRoleInCourse.ADMINISTRATOR);
+
+        tester = addCurrentUserHeader(tester, currentUser);
+
+        membershipRepository.save(CourseMembershipEntity.builder()
+                .userId(currentUser.getId())
+                .courseId(course.getId())
+                .role(UserRoleInCourse.ADMINISTRATOR)
+                .build());
+
+        final String query =
+                """
+                query($courseId: UUID!) {
+                    coursesByIds(ids: [$courseId]) {
+                        memberships {
+                            userId
+                            courseId
+                            role
+                        }
+                    }
+                }
+                """;
+
+        tester.document(query)
+                .variable("courseId", course.getId())
+                .execute()
+                .path("coursesByIds[0].memberships").entityList(CourseMembership.class).hasSize(1)
+                .path("coursesByIds[0].memberships[0].userId").entity(UUID.class).isEqualTo(currentUser.getId())
+                .path("coursesByIds[0].memberships[0].courseId").entity(UUID.class).isEqualTo(course.getId())
+                .path("coursesByIds[0].memberships[0].role").entity(UserRoleInCourse.class).isEqualTo(UserRoleInCourse.ADMINISTRATOR);
+    }
+
+    @Test
+    void testMembershipsFieldInCourseNoPermission(HttpGraphQlTester tester) {
+        final CourseEntity course = courseRepository.save(createTestCourse());
+
+        final LoggedInUser currentUser = userWithMembershipInCourseWithId(course.getId(),
+                LoggedInUser.UserRoleInCourse.STUDENT);
+
+        tester = addCurrentUserHeader(tester, currentUser);
+
+        membershipRepository.save(CourseMembershipEntity.builder()
+                .userId(currentUser.getId())
+                .courseId(course.getId())
+                .role(UserRoleInCourse.STUDENT)
+                .build());
+
+        final String query =
+                """
+                query($courseId: UUID!) {
+                    coursesByIds(ids: [$courseId]) {
+                        memberships {
+                            userId
+                            courseId
+                            role
+                        }
+                    }
+                }
+                """;
+
+        tester.document(query)
+                .variable("courseId", course.getId())
+                .execute().errors().expect(e ->
+                        e.getExtensions().get("exception").equals(NoAccessToCourseException.class.getSimpleName()));
```

Comment:
I think there is a utils method in gits common test, but maybe it doesnt fit here